### ISSUE TITLE
failing tests after new year 

### ISSUE THREAD

**Issue by erikcelander** [spencermountain/spacetime#362]
dst-off europe/london
dst-off atlantic/madeira
dst-now-off america/scoresbysund
dst-now-off atlantic/azores

**Comment by spencermountain** [comment]
hey Erik, thanks - I may need more details to reproduce this - what timezone are you in?  are they always failing for you?
you can see test are green on CI [3 days ago](https://github.com/spencermountain/spacetime/actions/runs/3809020348/jobs/6480073429)
cheers

**Comment by erikcelander** [comment]
yeah tests were passing for me a couple days ago as well, just noticed it today. 
Central European Time

**Comment by spencermountain** [comment]
hey Erik, i was able to reproduce this - it looks like [#235](https://github.com/spencermountain/spacetime/issues/235) again. 

I'm working on [a fix](https://github.com/spencermountain/spacetime/tree/cool-rewrite) but it's turning into a bigger re-write than i had planned.  maybe we should disable these tests for now, until that release is ready. it will be a month or two, most likely.
cheers
